sci-mathematics/octave/octave-9.2.0.ebuild

```diff
@@ -0,0 +1,227 @@
+
+EAPI=8
+
+inherit autotools flag-o-matic fortran-2 java-pkg-opt-2 pax-utils qmake-utils toolchain-funcs xdg
+
+DESCRIPTION="High-level interactive language for numerical computations"
+HOMEPAGE="https://www.gnu.org/software/octave/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~riscv ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="curl doc fftw +glpk gnuplot gui hdf5 imagemagick java json klu portaudio postscript +qhull +qrupdate readline sndfile +sparse spqr ssl sundials zlib"
+
+# Although it is listed in INSTALL.OCTAVE as a build tool, Octave runs
+# "makeinfo" from sys-apps/texinfo at runtime to convert its texinfo
+# documentation to text (see scripts/help/help.m).
+#
+# (un)zip isn't mentioned, but there's a test that uses it (bug #775254).
+#
+# The use of USE=imagemagick to pull in media-gfx/graphicsmagick is not
+# ideal, but both "graphicsmagick" and "imagemagick" are global USE
+# flags whose existing descriptions conflict with the obvious way we
+# would want to use them in octave. In any case, upstream doesn't really
+# support imagemagick, only graphicsmagick (bug 864785).
+COMMON_DEPS="
+	app-arch/bzip2
+	app-arch/unzip
+	app-arch/zip
+	app-text/ghostscript-gpl
+	sys-apps/texinfo
+	dev-libs/libpcre2
+	sys-libs/ncurses:=
+	sys-libs/zlib
+	virtual/blas
+	virtual/lapack
+	curl? ( net-misc/curl:= )
+	fftw? ( sci-libs/fftw:3.0= )
+	glpk? ( sci-mathematics/glpk:= )
+	gnuplot? ( sci-visualization/gnuplot )
+	hdf5? ( sci-libs/hdf5:= )
+	imagemagick? ( media-gfx/graphicsmagick:=[cxx] )
+	json? ( dev-libs/rapidjson )
+	klu? ( sci-libs/klu:= )
+	portaudio? ( media-libs/portaudio )
+	postscript? (
+		app-text/epstool
+		media-gfx/pstoedit
+		>=media-gfx/fig2dev-3.2.9-r1
+	)
+	gui? (
+		dev-qt/qtbase:6[gui,opengl,network,widgets]
+		media-libs/fontconfig:=
+		media-libs/freetype:=
+		virtual/glu
+		x11-libs/gl2ps:=
+		x11-libs/libX11:=
+		x11-libs/qscintilla:=[qt6]
+	)
+	qhull? ( media-libs/qhull:= )
+	qrupdate? ( sci-libs/qrupdate:= )
+	readline? ( sys-libs/readline:= )
+	sndfile? ( media-libs/libsndfile )
+	sparse? (
+		sci-libs/arpack:=
+		sci-libs/camd:=
+		sci-libs/ccolamd:=
+		sci-libs/cholmod:=
+		sci-libs/colamd:=
+		sci-libs/cxsparse:=
+		sci-libs/umfpack:=
+	)
+	spqr? ( sci-libs/spqr:= )
+	ssl? (
+		dev-libs/openssl:=
+	)
+	sundials? (
+		klu? ( >=sci-libs/sundials-4:=[sparse] )
+		!klu? ( >=sci-libs/sundials-4:= )
+	)
+"
+RDEPEND="${COMMON_DEPS}
+	java? ( >=virtual/jre-1.8:* )"
+DEPEND="${COMMON_DEPS}
+	java? ( >=virtual/jdk-1.8:* )"
+BDEPEND="
+	dev-util/gperf
+	virtual/pkgconfig
+	doc? (
+		dev-texlive/texlive-fontsrecommended
+		dev-texlive/texlive-plaingeneric
+		dev-texlive/texlive-metapost
+		virtual/latex-base
+	)
+	gui? ( dev-qt/qttools:6[linguist] )
+	qrupdate? ( app-misc/pax-utils )
+	sparse? ( app-misc/pax-utils )
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-5.1.0-pkgbuilddir.patch
+	"${FILESDIR}"/${PN}-9.2.0-omit-qtchooser-qtver.patch
```

Comment:
Is the patch still needed when only qt:6 is supported?

Reply:
indeed it is not needed anymore